byron-proxy/cfg/logging-validator.yaml

```diff
@@ -0,0 +1,40 @@
+# global filter; messages must have at least this severity to pass:
+minSeverity: Debug
+
+# these backends are initialized:
+setupBackends:
+  - EKGViewBK
+  - KatipBK
+
+# if not indicated otherwise, then messages are passed to these backends:
+defaultBackends:
+  - EKGViewBK
+  - KatipBK
+
+# if wanted, the GUI is listening on this port:
+# hasGUI: 18321
+
+# if wanted, the EKG interface is listening on this port:
+hasEKG: 12788
+
+# if wanted, the Prometheus interface is using this port:
+hasPrometheus: 12798
+
+# here we set up outputs of logging in 'katip':
+setupScribes:
+  - scKind: StdoutSK
+    scFormat: ScText
+    scName: stdout
+
+# if not indicated otherwise, then log output is directed to this:
+defaultScribes:
+  - - StdoutSK
```

Comment:
'- -' Intentional?

Reply:
Yes, for now. Otherwise proxy and validator have port conflicts. Will write a logConfigGenerator in a future iteration.